src/cljs/owlet_ui/components/search_bar.cljs

```diff
@@ -84,4 +86,8 @@
             :data-source suggestions-for-search
             :model search-model
             :placeholder "Search..."
-            :render-suggestion #(:term %)]]))}))
+            :render-suggestion (fn [t]
+                                 (let [platform-index (.indexOf platform-search-names (:term t))]
+                                   (if (>= platform-index 0)
+                                     (nth platform-names platform-index)
```

Comment:
@dviramontes there is an else case, but because it is the same as before it doesn't appear in the changes. Made the other changes you suggested